tutorials/benchmark_simulation.ipynb

```diff
@@ -0,0 +1,305 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.            # See section on scaling for explanation of the conversion.
</pre><p>This was not clear to me.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1071/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Sorry, forgot to delete after copying.